ebpf/Makefile

```diff
@@ -42,13 +43,15 @@ release: nightly bpf-linker
 
 .PHONY: nightly
 nightly: ## Add nightly toolchain (needed for eBPF)
-	rustup toolchain list | grep -qc 'nightly-x86_64-unknown-linux-gnu' || \
-		rustup toolchain install nightly-x86_64-unknown-linux-gnu --component rust-src
+	rustup toolchain list | grep -qc 'nightly-$(uname_m)-unknown-linux-gnu' || \
+		rustup toolchain install nightly-$(uname_m)-unknown-linux-gnu --component rust-src
 
 .PHONY: bpf-linker
 bpf-linker: ## Install bpf-linker (needed for eBPF)
-	cargo install --list | grep -qc 'bpf-linker' || \
-		cargo install bpf-linker
+	$(cargo) install --list | grep -qc 'bpf-linker' || \
+	if [ '$(uname_m)' = 'aarch64' ]; \
```

Comment:
In the main Makefile we use `ifeq`; I couldn't get that to work, though I think that would be nicer.

Reply:
x86_64 is the only officially supported arch as of right now so we don't need to worry about CI. I just wanted to keep the M1 as a viable dev environment, see as that's what I have :)